galaxy_ng/openapi/schema_generator.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from pulpcore.openapi import PulpSchemaGenerator
+
+log = logging.getLogger(__name__)
+
+
+class GalaxySchemaGenerator(PulpSchemaGenerator):
+
+    def convert_endpoint_path_params(self, path, view, schema):
+        """
+        Skip pulpcore's converting of urls into pulp hrefs
+        """
+        # need core tweaks
+        # if request and 'bindings' in request.params:
+        #     return super().convert_endpoint_path_params(path, view, schema)
```

Comment:
Not sure what needs to happen with the commented out block here

Reply:
Updated schema_generator to better explain what it's doing.

Added notes about convert_endpoint_path_params() needing to know if it is in 'binding' modes or not. To do that we need to change how pulpcore.openapi.PulpSchemaGenerator calls convert_endpoint_path_params() to pass in that info. Either by passing in the request object or some other var. If PulpSchemaGenerator does that, then the commented out code is how this method would fall back to it's super.